src/components/common/styles.js

```diff
@@ -133,6 +157,7 @@ export const StyledSubmit = styled.input`
   font-size: 1rem;
   background: #4CD2B2;
```

Comment:
Can we replace hardcoded hex codes with colors from the theme? If there's a color that's not in the theme yet, you can add it in App.js

Reply:
Fair point, I'll remove hardcoded colors and use what's in the theme.